Snakefile

```diff
@@ -117,64 +91,61 @@ rule get_merizo_clustering:
             > {output.output_file}
         """
 
-rule get_mdtask_clustering:
+rule build_pdbs:
     input:
-        correlation_file="output/correlation/{pdb_file}.txt"
+        topology="01_input/top/{protein_name}_complex.top",
+        trajectory="01_input/traj/{protein_name}_10-20ns_100snap.trr"
     output:
-        output_file="output/correlation/{pdb_file}.tsv"
-    shell:
-        """
-        python scripts/segment_by_correlation.py {input.correlation_file} {output.output_file}
-        """
-
-rule rename_top_file_to_prmtop:
-    input: "{path}.top"
-    output: "{path}.prmtop"
-    shell: "mv {input} {output}"
```

Comment:
We no longer need to rename the file, since I've stopped using `mdconvert` and I'm using mdanalysis to generate derivative files.